index.js

```diff
@@ -2,12 +2,7 @@
 module.exports = function (config) {
   var library = config.mock === false ?  // mock is on by default
       require('./trim.production') :
-      require('./trim.development');
+      require('./trim.development')
```

Comment:
I agree. You'll notice I didn't actually update the development codebase. Partially out of laziness and knowing that noone is actually using it, as well as disdain for the whole thing.